_includes/releases/v22.2/v22.2.0-beta.5.md

```diff
@@ -0,0 +1,65 @@
+## v22.2.0-beta.5
+
+Release Date: October 31, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-2-0-beta-5-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- Changefeed [Kafka sinks](../v22.2/changefeed-sinks.html#kafka) no longer automatically retries when emitting a message batch that gets rejected by the server. This is a temporary rollback of the functionality. [#90037][#90037]
+
+<h3 id="v22-2-0-beta-5-sql-language-changes">SQL language changes</h3>
+
+- Star expressions, e.g., `SELECT * FROM ...` are no longer allowed in statements in user-defined functions (UDFs). These were allowed in early betas of v22.2 from `v22.2.0-beta.1` to `v22.2.0-beta.4`, but have been disallowed because they do not behave correctly. Issue [#90080](https://github.com/cockroachdb/cockroach/issues/90080) tracks re-enabling star expressions in UDFs. [#90171][#90171]
```

Comment:
```suggestion
- Star expressions, e.g., `SELECT * FROM ...` are no longer allowed in statements in user-defined functions (UDFs). These were allowed in `v22.2.0-beta.1` to `v22.2.0-beta.4`, but have been disallowed because they do not behave correctly. Issue [#90080](https://github.com/cockroachdb/cockroach/issues/90080) tracks re-enabling star expressions in UDFs. [#90171][#90171]
```